ush/machine/wcoss2.yaml

```diff
@@ -44,10 +44,13 @@ cpl_aqm_parm:
   DCOMINcanopy: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/canopy
   DCOMINfire: /lfs/h2/emc/physics/noscrub/kai.wang/RAVE_fire/RAVE_NA_NRT
   DCOMINchem_lbcs: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/LBCS/AQM_NA13km_AM4_v1
-  DCOMINgefs: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/GEFS_DATA
+    #DCOMINgefs: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/GEFS_DATA
+  COMINgefs: /lfs/h1/ops/prod/com/gefs/v12.3 
   DCOMINpt_src: /lfs/h2/emc/physics/noscrub/Youhua.Tang/nei2016v1-pt/v2023-01-PT
   DCOMINairnow: /lfs/h1/ops/prod/dcom
-  AQM_AIRNOW_HIST_DIR: /lfs/h2/emc/physics/noscrub/jianping.huang/Bias_correction/aqmv7.0
+    #AQM_AIRNOW_HIST_DIR: /lfs/h2/emc/physics/noscrub/jianping.huang/Bias_correction/aqmv7.0e
```

Comment:
Remove commented-out lines

Reply:
Will do the global development code clean up later.